wordclock_interfaces/templates/app.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+	<head>
+	<title>Word Clock</title>
+	<link rel='stylesheet prefetch' href='https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons'>
+	<link rel='stylesheet prefetch' href='https://unpkg.com/vuetify@1.0.18/dist/vuetify.min.css'>
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
```

Comment:
Concerning the full-screen: You can add here:
`    <meta name="mobile-web-app-capable" content="yes">`
`    <meta name="apple-mobile-web-app-capable" content="yes">`
... to have the page in full-screen on android + ios devices.